projects/novo-elements/src/elements/form/controls/BaseControl.ts

```diff
@@ -16,158 +16,92 @@ export interface NovoGroupedControlConfig {
   initialValue?: {}[];
 }
 
-export interface NovoControlConfig {
-  validators?: Array<any>;
+class Control {
+  allowInvalidDate?: boolean;
+  appendToBody: boolean; // Deprecated;
```

Comment:
i'm concerned that this className conflicts with the actual `NovoConrol` do we want to call this `ControlConfig`?

Reply:
That sounds great, I'll rename it `ControlConfig` 👍